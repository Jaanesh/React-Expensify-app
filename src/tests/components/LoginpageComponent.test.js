import React from 'react';
import {shallow} from 'enzyme';
import {loginPageComponent} from '../../components/LoginpageComponent.js';

test('should render loginpage component',()=>{
      
       const wrapper=shallow(<loginPageComponent  startLogin={()=>{}} />);
       expect(wrapper).toMatchSnapshot();
});

/*  test('should call start login on button click',()=>{
     let startLogin=jest.fn();
     const wrapper=shallow(<loginPageComponent startLogin={startLogin} />);
     wrapper.find('button').simulate('click');
     expect(startLogin).toHaveBeenCalled();
}); */  