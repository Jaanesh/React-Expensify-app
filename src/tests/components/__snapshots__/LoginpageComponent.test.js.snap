// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render loginpage component 1`] = `
<loginPageComponent
  startLogin={[Function]}
/>
`;
